log/log_file.log

```diff
@@ -0,0 +1,8 @@
+
+
+2019-11-06 14:25:18 INFO  ServiceParsing:69 - Read number of items: 100000
```

Comment:
@TatianaPirch should we push it?

Reply:
I attached a log file to show how long it took to process csv-file